site/content/docs/testapps/altoroj.md

```diff
@@ -78,12 +109,24 @@ env:
         username: admin
 ```
 
+> [!NOTE]
+> There are no exclude paths added in the environment definition. Logout avoidance is used in the Traditional Spider job example below.
+> doLogin is left included as it is impacted by a SQLi vulnerability.
 
 ### Crawling
 
-Any of the ZAP Spiders can be used to crawl AltoroJ.
+Any of the ZAP Spiders can be used to crawl Testfire (AltoroJ).
+
+For the [Traditional Spider](/docs/desktop/addons/spider/) we recommend the following configuration:
 
-For the [Traditional Spider](/docs/desktop/addons/spider/) to be able to crawl the app you need to exclude the login and logout URLs from the context as above.
+```yaml
+- type: spider
+  parameters:
+    context: testfire
+    user: jsmith
```

Comment:
Better to use a single user to avoid confusion/mismatches.

Reply:
Oops, thanks